content/en/metrics/guide/different-aggregators-look-same.md

```diff
@@ -3,6 +3,7 @@ title: I'm switching between the sum/min/max/avg aggregators but the values look
 kind: faq
```

Comment:
I can't make a suggestion in the right place, but let's change the title to: "Switching Between the sum/min/max/avg Aggregators Doesn't Change the Value"